MapGameController.java

```diff
@@ -133,7 +136,7 @@ public void controlNPC(){
                     break;
             }
 
-            for(int Y =0;Y<15;Y++){
+            /*for(int Y =0;Y<15;Y++){
```

Comment:
Don't comment-out something unneeded, but just remove it.
We can go back with git whenever we need it.